code/modules/mob/living/carbon/human/species/station/machine_subspecies.dm

```diff
@@ -68,8 +68,8 @@
 	short_name = "ind"
 	name_plural = "Industrials"
 
-	brute_mod = 0.3
-	burn_mod = 1.2
```

Comment:
I really don't think burn mod should be increased, lasers are already overused as is. Infact i'd suggest dropping it down to 1

lasers will still be an advantage against them because of the brute mod.